app/helpers/time_helper.rb

```diff
@@ -24,6 +24,14 @@ def displayed_timezone(timezone)
   end
 
   def formatted_datetime(datetime)
+    return unless datetime
+
     datetime.strftime("%b %d %-l:%M %p")
   end
+
+  def formatted_datetime_with_break(datetime)
+    return unless datetime
+
+    content_tag(:span, datetime.strftime("%b %d")) + tag(:br) + content_tag(:span, datetime.strftime("%-l:%M %p"))
```

Comment:
I didn't know about `content_tag`. Cool!

https://apidock.com/rails/ActionView/Helpers/TagHelper/content_tag says it's "legacy syntax" and that https://apidock.com/rails/ActionView/Helpers/TagHelper/tag is preferred, e.g., `tag.span(datetime.strftime(...))`. I have no actual preference, but figured I'd mention it.